src/richie/plugins/glimpse/templates/richie/glimpse/glimpse.html

```diff
@@ -0,0 +1,38 @@
+{% load thumbnail %}{% spaceless %}
+
+{% with variant=variant|default:"card_square" %}
+
+    {% if instance.get_link %}
+        <a href="{{ instance.get_link }}"{% if instance.is_blank_target %} target="blank"{% endif %}
```

Comment:
I think target blank  takes a "_"? Apparently not safe to use without the following rel attribute:
```suggestion
        <a href="{{ instance.get_link }}"{% if instance.is_blank_target %} target="_blank" rel=”noopener noreferrer”{% endif %}
```